doc/sphinxext/gh_substitutions.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from docutils.nodes import reference
+from docutils.parsers.rst.roles import set_classes
+
+
+# adapted from
+# https://github.com/mne-tools/mne-python/blob/main/doc/sphinxext/gh_substitutions.py
+
+
+def gh_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
+    """Link to a GitHub issue."""
```

Comment:
Either call it `_gh_role` or write complete docstrings

Reply:
Thanks! Fixed in 1d6749e